docs/getting_started.rst

```diff
@@ -86,6 +86,9 @@ Use the ``HTTPAdapter`` argument ``pool_connections`` to set the number of
 urllib3 connection pools to cache, and the ``pool_maxsize`` argument to set the
 maximum number of connections to save in the pool.
 
+As long as the state of ``client`` does not changed it will be
+thread safe and accessible as a static variable to all threads.
+
```

Comment:
@emlaver sorry to be pedantic, it would be good to let users know how they can find out what changes the state of the client.